doc/governance.rst

```diff
@@ -53,15 +53,13 @@ the CLT itself.
 Current CLT members are:
 
  * Abhishek Lekshmanan <abhishek@suse.com>
- * Alfredo Deza <adeza@redhat.com>
  * Casey Bodley <cbodley@redhat.com>
  * Gregory Farnum <gfarnum@redhat.com>
  * Haomai Wang <haomai@xsky.com>
  * Jason Dillaman <dillaman@redhat.com>
  * Josh Durgin <jdurgin@redhat.com>
  * João Eduardo Luis <joao@suse.de>
  * Ken Dreyer <kdreyer@redhat.com>
- * Lenz Grimmer <lgrimmer@suse.com>
```

Comment:
i think @epuertat is the component lead of dashboard team now.

Reply:
I've added him.

Reply:
Thank you, @tchaikov !